aws-iot-accountauditconfiguration/aws-iot-accountauditconfiguration.json

```diff
@@ -0,0 +1,114 @@
+{
+  "typeName": "AWS::IoT::AccountAuditConfiguration",
+  "description": "Configures the Device Defender audit settings for this account. Settings include how audit notifications are sent and which audit checks are enabled or disabled.",
+  "sourceUrl": "https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-iot.git",
+  "definitions": {
+    "AuditCheckConfiguration": {
+      "description": "The configuration for a specific audit check.",
+      "type": "object",
+      "properties": {
+        "Enabled": {
+          "description": "True if the check is enabled.",
+          "type": "boolean"
+        }
+      }
+    },
+    "AuditNotificationTarget": {
+      "type": "object",
+      "properties": {
+        "TargetArn": {
+          "description": "The ARN of the target (SNS topic) to which audit notifications are sent.",
+          "type": "string",
+          "maxLength": 2048
```

Comment:
Minor: Would it be better to keep this consistent with our CP definition:
https://code.amazon.com/packages/IotSepioServiceModel/blobs/df8766043d97ea106eaa742aa63f09c6ec6b5779/--/model/common-types.xml#L186